upup/pkg/fi/cloudup/crictl.go

```diff
@@ -0,0 +1,40 @@
+package cloudup
+
+import (
+	"fmt"
+	"net/url"
+
+	"k8s.io/kops/pkg/apis/kops"
+	"k8s.io/kops/pkg/assets"
+	"k8s.io/kops/util/pkg/architectures"
+	"k8s.io/kops/util/pkg/hashing"
+)
+
+const (
+	crictlAssetUrlAmd64 = "https://github.com/kubernetes-sigs/cri-tools/releases/download/v1.29.0/crictl-v1.29.0-linux-amd64.tar.gz"
+	crictlAssetUrlArm64 = "https://github.com/kubernetes-sigs/cri-tools/releases/download/v1.29.0/crictl-v1.29.0-linux-arm64.tar.gz"
```

Comment:
I'm wondering how to update this version when a new version is released. Do you have any ideas?

Reply:
Great question; we have done it by hand so far.  The other problem is that it would be nice if we "hard-coded" the sha somehow, both for security and to avoid having to download the sha file "every time".  One thing we could do is put it into a yaml file like the channels file(s), but find a format that dependabot could use (or just create our own dependabot like tool for our yaml file, if we can't fit it into dependabot).  We could either download that channels file on demand, or we could use `go:embed` (or both!)

As this is not something this PR is introducing, I don't consider it a blocker for this PR.  But cc @hakman to see if he has any thoughts here.